uaclient/schemas.py

```diff
@@ -0,0 +1,301 @@
+contract_schema = {
```

Comment:
Let's align the name of this with its purpose
```suggestion
machine_token_file_schema = {
```